resources/zib/zib-GlasgowComaScale.xml

```diff
@@ -0,0 +1,256 @@
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="281396004" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:gcsMotor.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="GCS_Motor" />
+      <definition value="Best motor response to a stimulus. The element is mandatory, but does not require a numerical value. If no value is given, the reason for this is to be provided in the ConditionsDuringMeasurement concept." />
+      <alias value="GCS_Motor" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.121.11.5--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-glasgowcomascale-v3.2-2020EN" />
+        <map value="NL-CM:12.8.4" />
+        <comment value="GCS_Motor" />
+      </mapping>
+    </element>
+    <element id="Observation.component:gcsVerbal">
+      <path value="Observation.component" />
+      <sliceName value="gcsVerbal" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:gcsVerbal.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="281397008" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:gcsVerbal.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="GCS_Verbal" />
+      <definition value="Best verbal response to a stimulus. The element is mandatory, but does not require a numerical value. If no value is given, the reason for this is to be provided in the ConditionsDuringMeasurement concept." />
+      <alias value="GCS_Verbal" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.121.11.6--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-glasgowcomascale-v3.2-2020EN" />
+        <map value="NL-CM:12.8.6" />
+        <comment value="GCS_Verbal" />
+      </mapping>
+    </element>
+    <element id="Observation.component:conditionsDuringMeasurement">
+      <path value="Observation.component" />
+      <sliceName value="conditionsDuringMeasurement" />
+    </element>
+    <element id="Observation.component:conditionsDuringMeasurement.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="271000124103" />
```

Comment:
Add a comment w.r.t. this code (see the section on missing definition codes in https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4#For_zibs_specifically).

Reply:
I'm wondering if we've always added this comment in the past; maybe we should do an additional check on all profiles.